src/neo4j_genai/retrievers/hybrid.py

```diff
@@ -83,6 +86,9 @@ def search(
 
         search_query = get_search_query(SearchType.HYBRID, self.return_properties)
 
+        logger.debug("HybridRetriever Cypher parameters", parameters)
```

Comment:
Another point: we need to use placeholder in the first string for the other parameters to be used (sorry missed that during first read). I.e.:
``` 
logger.debug("HybridRetriever Cypher parameters %s", parameters)
```